modules/tinymce/src/core/main/ts/focus/EditorFocus.ts

```diff
@@ -119,6 +119,11 @@ const focusEditor = (editor: Editor) => {
     normalizeSelection(editor, rng);
   }
 
+  // This should be safe as the the focusin listener calls editor.focus(true) which means the focusEditor function is not called
+  // TODO: Should this also include the last blurredEditor?
+  // TODO: Need some tests for this
+  editor.fire('focus');
```

Comment:
Actually, why was this added? This almost definitely seems incorrect as we'll end up with the focus event firing twice.

Reply:
Have added an `EditorFocus` event that is fired on `editor.focus()` since listening to `focus` is not sufficient to detect all focuses of the editor.